huaweicloud/resource_huaweicloud_rds_read_replica_instance.go

```diff
@@ -0,0 +1,390 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/rds/v3/instances"
+)
+
+func resourceRdsReadReplicaInstance() *schema.Resource {
+
+	return &schema.Resource{
+
+		Create: resourceRdsReadReplicaInstanceCreate,
+		Read:   resourceRdsReadReplicaInstanceRead,
+		Delete: resourceRdsReadReplicaInstanceDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
seems we can not import it as primary instance id was not exist in Read method.

Reply:
Emmm, but we'll know the id of the read replica instance after creating it, then could we import it?

Reply:
I have modified the importer of the resource. User shoule provide `primary_instance_id` and `id` before importing a read replica instance resource.